### PR TITLE
Kotlin-1.4.0

### PR DESCRIPTION
**Pull request by antonioseric**
I manage to make it compile but i didn't test it... 
Issue: #965

### CODE REVIEW

**Comment by Nikituh** [comment]
I would also very much like to have this released. I'll also note that I did build it locally myself, and had it tested, and I'd say ship it 🚢

File: Android/Kotlin/source/Xamarin.Kotlin.StdLib/Transforms/Metadata.xml [hunk 1]
```diff
@@ -95,13 +88,15 @@
...
 
   <attr path="/api/package[@name='kotlin.ranges']/class[@name='CharRange']/method[@name='contains' and count(parameter)=1 and parameter[1][@type='char']]/parameter[1]" name="managedType">Java.Lang.Character</attr>
   <attr path="/api/package[@name='kotlin.ranges']/class[@name='IntRange']/method[@name='contains' and count(parameter)=1 and parameter[1][@type='int']]/parameter[1]" name="managedType">Java.Lang.Integer</attr>
   <attr path="/api/package[@name='kotlin.ranges']/class[@name='LongRange']/method[@name='contains' and count(parameter)=1 and parameter[1][@type='long']]/parameter[1]" name="managedType">Java.Lang.Long</attr>
-  <attr path="/api/package[@name='kotlin.ranges']/class[@name='UIntRange']/method[starts-with(@name,'contains-') and count(parameter)=1 and parameter[1][@type='uint']]/parameter[1]" name="managedType">Kotlin.UInt</attr>
-  <attr path="/api/package[@name='kotlin.ranges']/class[@name='ULongRange']/method[starts-with(@name,'contains-') and count(parameter)=1 and parameter[1][@type='ulong']]/parameter[1]" name="managedType">Kotlin.ULong</attr>
```
Comment (mattleibow) [inline Android/Kotlin/source/Xamarin.Kotlin.StdLib/Transforms/Metadata.xml:104]: With the new generator, `UInt` and `ULong` are no longer a thing. The types exist, but there is no way to get one of them. Everything is the nice `uint` and `ulong`.

File: Android/Kotlin/source/Xamarin.Kotlin.StdLib/Transforms/Metadata.xml [hunk 2]
```diff
@@ -163,6 +169,8 @@
   <remove-node path="/api/package[@name='kotlin.text']/class[@name='StringsKt__StringsKt']/method[@name='trimStart' and count(parameter)=2 and parameter[1][@type='java.lang.String'] and parameter[2][@type='char...']]" />
   <remove-node path="/api/package[@name='kotlin.text']/class[@name='StringsKt__StringsKt']/method[@name='trimStart' and count(parameter)=2 and parameter[1][@type='java.lang.String'] and parameter[2][@type='kotlin.jvm.functions.Function1&lt;? super java.lang.Character, java.lang.Boolean&gt;']]" />
 
+  <remove-node path="/api/package[@name='kotlin.time']/class[@name='Duration']/method[@name='div-LRDsOJo' and count(parameter)=2 and parameter[1][@type='double'] and parameter[2][@type='double']]" />
```
Comment (mattleibow) [inline Android/Kotlin/source/Xamarin.Kotlin.StdLib/Transforms/Metadata.xml:172]: This is a special case where the second parameter is really a `Duration` - a magical type that is a Kotlin-ified `double`. Just like how `ULong` is a `ulong` and you can't get one.

**Review by mattleibow — APPROVED** [review]
Thanks for the PR! I had to change things for the new generator. Or rather, stop doing bad things and just let the new generator shine!

**Comment by mattleibow** [comment]
It is coming out soon folks! Sorry it took so long!

**Comment by mattleibow** [comment]
Just letting everyone know that both v1.4 versions are out!

**Pull request merged**